WalletWasabi/Wallets/Wallet.cs

```diff
@@ -233,7 +233,7 @@ public int GetPrivacyPercentage()
 	{
 		var currentPrivacyScore = Coins.Sum(x => x.Amount.Satoshi * Math.Min(x.HdPubKey.AnonymitySet - 1, x.IsPrivate(AnonScoreTarget) ? AnonScoreTarget - 1 : AnonScoreTarget - 2));
 		var maxPrivacyScore = Coins.TotalAmount().Satoshi * (AnonScoreTarget - 1);
-		int pcPrivate = maxPrivacyScore == 0M ? 100 : (int)(currentPrivacyScore * 100 / maxPrivacyScore);
+		int pcPrivate = maxPrivacyScore == 0M ? 0 : (int)(currentPrivacyScore * 100 / maxPrivacyScore);
```

Comment:
Could someone from the @zkSNACKs/code-team verify that this doesn't break anything in CJ code? @adamPetho @molnard

Reply:
Thanks!